deeppavlov/models/multitask_bert/multitask_bert.py

```diff
@@ -1121,32 +1121,3 @@ def __call__(self, *args, **kwargs) -> List[Any]:
         """
         res = self.mt_bert.call(args, kwargs, task_names=self.task_names, in_distribution=self.in_distribution)
         return res
-
-
-@register("input_splitter")
-class InputSplitter:
```

Comment:
Are you sure that this deletion won't impact the existing multitask model?

Reply:
Also, file __inti__.py needs to be renamed

Reply:
> Are you sure that this deletion won't impact the existing multitask model?

Removing this should not have an impact on the multitask model because it will also be referenced by its registry name `'input_splitter'`
I have extracted this out into its own component.

Reply:
> Also, file **inti**.py needs to be renamed

It is already `__init__.py` , what should I rename it to?

Reply:
No, it is __inti__.py, not __init__.py

Reply:
Renamed the file.